include/pybind11/detail/internals.h

```diff
@@ -419,11 +425,24 @@ PYBIND11_NOINLINE internals &get_internals() {
     } gil;
     error_scope err_scope;
 
-    PYBIND11_STR_TYPE id(PYBIND11_INTERNALS_ID);
-    auto builtins = handle(PyEval_GetBuiltins());
-    if (builtins.contains(id) && isinstance<capsule>(builtins[id])) {
-        internals_pp = static_cast<internals **>(capsule(builtins[id]));
+    const char *id_cstr = PYBIND11_INTERNALS_ID;
+    PYBIND11_STR_TYPE id(id_cstr);
```

Comment:
I don't think we still need this macro anymore for compat.
```suggestion
    str id(id_cstr);
```

Reply:
Yes, this is still provided for backward compatibility only.